service.yml

```
@@ -58,3 +58,7 @@ semaphore:
             options:
               - 'deb'
               - 'ubi'
+code_artifact:
+  enable: true
+  package_paths:
+    - maven-snapshots/maven/io.confluent/cp-schema-registry
```

This doesn't look right. I think it should be:
```suggestion
    - maven-snapshots/maven/io.confluent.schema-registry-images/schema-registry-images-parent
```

---

Got it, just committed that change